README.md

```diff
@@ -19,9 +19,19 @@ let analyzer = new Analyzer({
   urlLoader: new FSUrlLoader(pathToPackageRoot),
```

Comment:
Perhaps rather than making this a magic variable name not defined in the scope of this example (`pathToPackageRoot`), this could be a string similar to the path passed to analyzer.analyze?

e.g. `urlLoader: new FSUrlLoader('/path/to/package/root')`